examples/interactivity/lines/feature-click.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
```

Comment:
We've changed the examples from their directory like 10 times in 4 weeks, I really don't care, but this is much noise in the PRs, it makes impossible to check for actual change on the examples and it doesn't get along with the browser history, please, commit to one path and stick to it in following PRs. Thank you

Reply:
Yes. The issue with that is that we have the same feature events with different geometries so they are now organized by geometry. Agree. We should move files in other PRs to avoid this diff issue :wink:

Reply:
:+1: